### PR TITLE
FIPS: fix openssl tests for MicroOS

### PR DESCRIPTION
**Pull request by jlausuch**
https://progress.opensuse.org/issues/125558

VR: https://openqa.opensuse.org/tests/3162983

I created https://bugzilla.opensuse.org/show_bug.cgi?id=1209053 to label the failing job

### CODE REVIEW

File: data/journal_check/bug_refs.json [hunk 1]
```diff
@@ -437,5 +437,12 @@
             "sle": ["15-SP4"]
         },
         "type": "ignore"
+    },
+    "Failed to start OpenSSH Daemon": {
```
Comment (Vogtinator) [inline data/journal_check/bug_refs.json:441]: This needs some explanation - where does this happen, why and why can it be ignored?

Comment (jlausuch) [reply data/journal_check/bug_refs.json]: https://bugzilla.opensuse.org/show_bug.cgi?id=1209063

**Comment by jlausuch** [comment]
@Vogtinator do you agree ? Or shall I include new section in journal_check with [bsc#1209063](https://bugzilla.opensuse.org/show_bug.cgi?id=1209063) ?
In any case, we will need to label the job with [bsc#1209053](https://bugzilla.opensuse.org/show_bug.cgi?id=1209053)

**Comment by Vogtinator** [comment]
IMO labelling that as known failure is ok.

**Comment by jlausuch** [comment]
Latest VR: https://openqa.opensuse.org/tests/3163183

**Comment by jlausuch** [comment]
Ok to merge?

**Review by Vogtinator — APPROVED** [review]

**Pull request merged**